floodviz/tests/test_utils.py

```diff
@@ -0,0 +1,407 @@
+import unittest
+import requests_mock
+from floodviz.utils import parse_rdb
+
+
+class TestPeakUtilsRDBparse(unittest.TestCase):
```

Comment:
The convention for naming tests classes is TestCamelCasedMethodName so this should be TestParseRdb.

The tests need to be updated. There are properties on self that are not used in the tests. I would recommend simplifying your self.response to be a few lines of comments, a few columns, and then three or four data lines. It does not need to mimic a real response. You will also want to have a test where the  response includes no data lines.